Python/ast.c

```diff
@@ -903,25 +907,37 @@ static const char * const FORBIDDEN[] = {
     "None",
     "True",
     "False",
+    "__debug__",
     NULL,
 };
 
+static int
+is_forbidden_name(const char *name)
```

Comment:
Can we not duplicate this by having ast_for_call turn the name into an identifier and passing it to `forbidden_name`?